src/Data/Product/Relation/Pointwise/NonDependent.agda

```diff
@@ -253,6 +253,22 @@ module _ {a b} {A : Set a} {B : Set b} where
 ------------------------------------------------------------------------
 -- Some properties related to "relatedness"
 
+<_,_>ₛ : ∀ {a₁ a₂ b₁ b₂ c₁ c₂}
+  {A : Setoid a₁ a₂} {B : Setoid b₁ b₂} {C : Setoid c₁ c₂} →
+  A ⟶ B → A ⟶ C → A ⟶ (B ×ₛ C)
```

Comment:
Really minor point, but would it be possible to add some bracketing to make this type clearer?

Reply:
Done!